Multiple event subscribing (#10)
wso2-extensions/esb-inbound-salesforce: README.md

```
* connection.salesforce.replay: replay **enable** or **disable**. Enabling this will read the event ID stored in the Registry DB or from the text file stored in the local machine.
* connection.salesforce.EventIDStoredFilePath:

>     - when replay is enabled, do not define any value for this property (i.e., keep it blank), to replay from the last event ID stored in the config Registry DB (property- name of the salesforce object(Follow bellow example) resource path:connector/salesforce/event).

Reviewer: ```suggestion
    - when replay is enabled, do not define any value for this property (i.e., keep it blank), to replay from the last event ID stored in the config Registry DB (property- name of the Salesforce object (follow the example below for more understanding) resource path - connector/salesforce/event).
```

Author: fixed in #11

    - when replay is enabled, specify the directory path of a text file to start replaying from the event ID stored in it. 
    ```

>     Ex: Salesforce object is /topic/reading__e then property name is "reading__e".

Reviewer: ```suggestion
    E.g., if the Salesforce object is `/topic/reading__e`, the property name is `reading__e`.
```

E.g. is a short form for a latin term called exempli gratia. This term translates to "for example". We cannot use different abbreviations like "Ex." for this.

Author: fixed in #11

Click **Execute**.
The Event will be triggered in the ESB in real time.

```

> To subscribe to multiple platform events or push topics add the separate inbound endpoit with corresponding sequences for push topics and platform events. Make sure to create resource in the config registry DB to for each object to store event id when replay option need to use.

Author: fixed in #11
